OctaneSdk/Services/EntityService.cs

```diff
@@ -440,5 +450,20 @@ public CommitPattern ValidateCommitMessage(IRequestContext context, string commi
 
             return jsonSerializer.Deserialize<CommitPattern>(response.Data);
         }
+
+        /// <summary>
+        /// Return all possible transitions for a given entity type
+        /// </summary>
+        public async Task<EntityListResult<Transition>> GetPosibbleTransitionsForEntityType(IRequestContext context, string entityType)
```

Comment:
1.Errata - GetPosibbleTransitionsForEntityType =>GetPossibleTransitionsForEntityType
2.Why we need "Possible" in name of method, is it possible to call it GetTransitionsForEntityType ?

Reply:
removed "possible" from the name